install_files/ansible-base/roles/common/tasks/apt_upgrade.yml

```diff
@@ -33,22 +42,22 @@
     # If we're connecting to an Onion URL, then it's over Tor.
     - (ansible_host|default(ansible_host)).endswith('.onion')
 
-- name: Perform safe upgrade to ensure all the packages are updated.
+- name: Perform dist upgrade to ensure all the packages are updated.
   apt:
-    upgrade: safe
+    upgrade: dist
+    install_recommends: no
```

Comment:
It is not, it's completely superfluous. I started by adding `install_recommends: no`, then realized it would take a lot of edits. So provided that the `80securedrop` file gets written before the upgrade, as here, then we can remove the install_recommends parameter.